dali/python/nvidia/dali/external_source.py

```diff
@@ -520,6 +520,20 @@ class ExternalSource():
         Python process, but due to their state it is not possible to calculate more
         than one batch at a time.
 
+`repeat_last` : bool, optional, default = False
+    .. note::
+        This is an advanced setting that is usable mainly with Triron inference server
+        with decoupled models.
+
+    Normally, ``external_source`` consumes its and expects new ones to be fed in the upcoming
+    iteration. Setting ``repeat_last=True`` changes this behavior so that ``external_source``
+    will detect that no new data was fed between the previous pipeline run and the currnet one
+    and will refeed the most recently data.
+
+    Setting ``repeat_last`` to `True` only makes sense in "push" mode, i.e. when the data is
+    actively provided by the user via a call to ``feed_input`` and is incompatible with specifying
```

Comment:
Technically, _the option_ is compatible, because you can set it to `False` explicitly. Setting it to `True` isn't and I wanted to capture that. I'll use different wording:
```suggestion
    actively provided by the user via a call to ``feed_input``. Enabling this option is incompatible with specifying
```